project/ScalaVersion.scala

```diff
@@ -1,10 +1,10 @@
 sealed abstract class MajorScalaVersion
-case object Scala29 extends MajorScalaVersion
 case object Scala210 extends MajorScalaVersion
+case object Scala211 extends MajorScalaVersion
 
 object ScalaVersion {
   def v(implicit sv: String) =
-    if(sv startsWith "2.9.") Scala29
-    else if(sv startsWith "2.10.") Scala210
+    if(sv startsWith "2.10.") Scala210
```

Comment:
This is unnecessary now that I've removed the casing on rojoma-json version. Mind if I remove?